tests/test_image_types.py

```diff
@@ -0,0 +1,337 @@
+import shutil
+
+import wagtail_factories
+
+from django.conf import settings
+from django.test import override_settings
+from test_grapple import BaseGrappleTest, BaseGrappleTestWithIntrospection
+from wagtail import VERSION as WAGTAIL_VERSION
+from wagtail.images import get_image_model
+
+from grapple.types.images import rendition_allowed
+
+
+class ImageTypesTest(BaseGrappleTestWithIntrospection):
+    @classmethod
+    def setUpTestData(cls):
+        cls.image_model = get_image_model()
+        cls.example_image = wagtail_factories.ImageFactory(title="Example Image")
+        cls.example_image.full_clean()
+        cls.example_image.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+        shutil.rmtree(settings.MEDIA_ROOT, ignore_errors=True)
+
+    def test_properties_on_saved_example_image(self):
+        example_img = self.image_model.objects.first()
+
+        self.assertEqual(example_img.id, self.example_image.id)
+        self.assertEqual(example_img.title, "Example Image")
+
+    def test_query_url_field(self):
+        query = """
+        {
+            images {
+                id
+                url
+                src
+            }
+        }
+        """
+
+        executed = self.client.execute(query)
```

Comment:
As we're not checking `"images" in executed["data"]` or any other keys, it might be nice to change this to:

```suggestion
        executed = self.client.execute(query)["data"]["images"][0]
```

To make your asserts simpler, e.g.:

```
self.assertEqual(executed["id"], str(self.example_image.id))
```